### ISSUE TITLE
Forced uppercase → lowercase conversion (inconsistent with Anki Desktop)

### ISSUE THREAD

**Issue by boolbag** [ankidroid/Anki-Android#7059]
On a specific card, the string `DLE` is automatically converted to `dle` on Ankidroid when the card is shown.
On other cards, this does not happen: `DLE` is shown as `DLE`, consistent with Anki Desktop.

Running 2.14alpha1

###### Reproduction Steps

Import this text card, with two fields separated with semi-colons:

```
The previous paragraph is a single statement, very cumbersome, which can be separated into four to eliminate some ambiguities.<br><br>(without <i>declaración, afirmación</i>, three options for <i>cumbersome</i>);El párrafo anterior es un sólo enunciado, muy farragoso, que puede separarse en cuatro para eliminar algunas ambigüedades.<br><br>/<i>engorroso, laborioso, trabajoso</i><br><i>un fárrago</i>: DLE → (i) <i>Cúmulo de ideas o expresiones confusas, inconexas o superfluas</i>, (ii) <i>Montón de cosas desordenadas.</i><br><i>enunciar</i>: to state, put forward<br>(María Cristina Dutto et al., <i>Palabras más, palabras menos</i>);
```
###### Expected Result

The text `DLE` should show as DLE (uppercase)

###### Actual Result

It shows as `dle`

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

###### Research
*Enter an [x] character to confirm the points below:*

- [x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [ x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x ] I have searched for similar existing issues here and on the user forum
- [x ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by mikehardy** [comment]
what in the world!? :sweat_smile: 

I'm not aware of any casing changes we would (or should!) do on user generated content. Makes no sense!

I assume all the other capitalized items in the card are fine? And it is capitalized in the database? What does it look like in Card Browser? And it is still DLE in Note editor?

**Comment by david-allison-1** [comment]
@boolbag could you provide a deck export which exhibits this problem?

**Comment by boolbag** [comment]
Thank you both for you replies. Obviously this does not bother me, just thought I'd report it as it seems odd.

First let me reply to @mikehardy : here are some screenshots in the browser, editor, and review. Sorry about not responding faster, on the road at the moment. 
Are all other capitalized cards fine? I don't know, there are many. Did a filtered deck for `DLE`, reviewing 20 such cards all were fine, but there are more. 

Wishing you all a fun weekend.

![browser](https://user-images.githubusercontent.com/7551671/92989351-16bdc100-f4d4-11ea-8431-3bee15b7116e.jpg)
![editor](https://user-images.githubusercontent.com/7551671/92989352-17eeee00-f4d4-11ea-9499-ebc52255b487.jpg)
![review](https://user-images.githubusercontent.com/7551671/92989354-18878480-f4d4-11ea-8762-6d25caa7ccc2.jpg)

**Comment by mikehardy** [comment]
That's just bizarre, thanks for the extra info

**Comment by david-allison-1** [comment]
@boolbag Edit the card, press "Cards" and enter: `.night_mode {}` in the CSS section.

**Comment by mikehardy** [comment]
I'll do a beta3 in a little while and google should dish it out in a few hours, it will have the fix for this. Thanks for reporting!

**Comment by boolbag** [comment]
Excited to try it out when the new alpha is released.
If I understand the hints in the thread, the `color: DLE` bit was being parsed as if it were css?
Must have been a fun one to troubleshoot.
Thank you very much.

**Comment by david-allison-1** [comment]
> Excited to try it out when the new alpha is released.
> If I understand the hints in the thread, the `color: DLE` bit was being parsed as if it were css?
> Must have been a fun one to troubleshoot.
> Thank you very much.

Wasn't bad at all thanks to your screenshot!

Pretty much, we tried to invert the color "DLE", and if it failed, we replaced it with the lowercase value rather than the original

https://github.com/david-allison-1/Anki-Android/blob/cea065fd60d6311d9dbc5a0c34ccbd055eed4ebe/AnkiDroid/src/main/java/com/ichi2/themes/HtmlColors.java#L41

**Comment by boolbag** [comment]
Confirming fixed on 2.14alpha2. 
Thank you!
: ) : ) : )
